pom.xml

```diff
@@ -91,6 +91,7 @@
         <apacheatlas.version>2.0.0</apacheatlas.version>
         <slf4j.version>1.7.25</slf4j.version>
         <jackson.version>2.9.9</jackson.version>
+        <jackson.databind.version>2.9.9.3</jackson.databind.version>
```

Comment:
this relates to another fix, but mostly is probably a reminder to us all we need to try and merge PRs promptly and in the right order to try and keep the linkages clean :-)